File: Language_Coder/src/function/Code586.java
package function;

import java.util.Scanner;

public class Code586 {

	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);

		int num1, num2;
		num1 = scan.nextInt();
		num2 = scan.nextInt();

		System.out.printf("(%d - %d) ^ 2 = %d\n", num1, num2, (int) Math.pow(num1 - num2, 2));
		System.out.printf("(%d + %d) ^ 3 = %d", num1, num2, (int) Math.pow(num1 + num2, 3));

	}

}

File: Language_Coder_Cpp/Code586/Code586.cpp
// Code586.cpp

#include <iostream>
#include <cmath>
using namespace std;

void func1(int num1, int num2) {
	cout << "(" << num1 << " - " << num2 << ") ^ " << 2 << " = " << (int)pow((num1 - num2), 2) << endl;
}

void func2(int num1, int num2) {
	cout << "(" << num1 << " + " << num2 << ") ^ " << 3 << " = " << (int)pow((num1 + num2), 3) << endl;
}

int main()
{
	int num1, num2;
	cin >> num1 >> num2;
	func1(num1, num2);
	func2(num1, num2);
}